listenbrainz/webserver/views/player.py

```diff
@@ -43,17 +43,21 @@ def load():
             error_msg="'Listens' array must have one or more items."
         )
 
-    user_data = {
+    current_user_data = {
         "id": current_user.id,
         "name": current_user.musicbrainz_id,
         "auth_token": current_user.auth_token,
     }
     spotify_data = spotify.get_user_dict(current_user.id)
     props = {
-        "user": user_data,
+        "user": {
+            "id": current_user.id,
+            "name": current_user.musicbrainz_id,
+        },
```

Comment:
This feels like a bit of a hack (the Recommendations component is supposed to show recs *for a user*, whereas the player isn't), but I don't see an actual issue anywhere and there's no easy way to make feedback work for the user on this page without refactoring the Recommendations component.

I think it would be good to add a comment here in player.py explaining why we are manually making user equal to current_user, so that the code doesn't get copied in another page or"fixed" without warning.

Reply:
hmmm, yeah, the recommendation component expects `user` and `current user` as keys for its working, it is indeed a hack, adding comment makes sense, thanks for pointing out :)